tests/acceptance/features/component/environment.py

```diff
@@ -138,8 +138,11 @@ def after_scenario(context, scenario):
 
     # Close all RabbitMQ consumers (if initiated)
     for consumer in context.rabbitmq_consumer_list:
-        consumer.stop()
-        consumer.close_connection()
+        try:
+            consumer.stop()
+            consumer.close_connection()
+        except Exception:
+            __logger__.warn("Rabbitmq consumer was already stopped")
```

Comment:
if the consumer was previously stopped you do not need to close the connection?

Or just the message should be "Rabbitmq consumer was already stopped and closed"